opta/commands/config.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional
```

Comment:
we need unit test to cover this class similar to the other commands.
you can look at `test_generate_terraform` it's the latest we added